internal/cache/cache.go

```diff
@@ -13,13 +13,14 @@ import (
 	"log/slog"
 	"os"
 	"path/filepath"
+	"syscall"
 	"time"
 )
 
 type RefreshReadWriter interface {
-	Read(any) error
-	Write(any) error
-	Refresh(time.Time)
+	Read(d any) error
+	Write(d any) error
+	Refresh(t time.Time)
 	RefreshedAt() time.Time
 }
 
```

Comment:
Did a linter suggested you to add variable name here? If so, which one?

I'm curious about adding it to my own settings

Reply:
I think it was https://github.com/macabu/inamedparam